week-1/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function here
-
+function createLongGreeting(name, age) {
+  return "Hello, my name is " + name + " and I'm " + age + " years old";
+}
 const greeting = createLongGreeting("Daniel", 30);
```

Comment:
Good use of const, make sure you know the difference between const and var 👍

Reply:
Thanks @AlexanderBraithwaite . Yes I will do  +1.